pilot/pkg/bootstrap/server.go

```diff
@@ -1307,14 +1317,13 @@ func (s *Server) initWorkloadTrustBundle(args *PilotArgs) error {
 	return nil
 }
 
-// isCADisabled returns whether CA functionality is disabled in istiod.
-// It returns true only if istiod certs is signed by Kubernetes or
-// workload certs are signed by external CA
-func (s *Server) isCADisabled() bool {
+// isK8SSigning returns whether K8S is used to sign certs instead of private keys known by Istiod
```

Comment:
Can you add comment this can be not k8s itself, but other provider

Reply:
Good point - it'll be less confusing when we remove the 'k8s itself' case completely.
Will try to find better wording.